integration_tests/cli/auth-demo.sh

```diff
@@ -231,12 +244,25 @@ check_response_and_report "$result"  '"info": "Alice proposes to revoke account"
 
 test_divider
 
+echo "Get list revoked account must be exist"
+result=$(dcld query auth list-revoked-account)
+check_response "$result" "\[\]"
+response_does_not_contain "$result" "\"address\": \"$user_address\""
+
+test_divider
+
 echo "Bob approves to revoke account for $user"
 result=$(echo $passphrase | dcld tx auth approve-revoke-account --address="$user_address" --from bob --yes)
 check_response "$result" "\"code\": 0"
 
 test_divider
 
+echo "Get list revoked account must be exist"
```

Comment:
Should be `Message should be Get all revoked accounts. $user account in the list` to be consistent

Reply:
Done! ✅